route_following_plugin/include/route_following_plugin.h

```diff
@@ -64,6 +65,18 @@ namespace route_following_plugin
          */
         void initialize();
 
+        // wm listener pointer and pointer to the actual wm object
+        std::shared_ptr<carma_wm::WMListener> wml_;
+        carma_wm::WorldModelConstPtr wm_;
+
+        /**
+         * \brief Compose UpcomingLaneChangeStatus msg from given starting and ending lanelets
```

Comment:
This method comment should specify that this method will only function correctly if the two provided lanelets are actually forming a lane change. This is an important assumption to document